llvm/unittests/Support/ParallelTest.cpp

```diff
@@ -99,7 +99,12 @@ TEST(Parallel, TaskGroupSequentialFor) {
   {
     parallel::TaskGroup tg;
     for (size_t Idx = 0; Idx < 500; Idx++)
-      tg.spawn([&Count, Idx]() { EXPECT_EQ(Count++, Idx); }, true);
+      tg.spawn(
```

Comment:
This TaskGroupSequentialFor test  is not necessary at all as "sequential" is removed.

Reply:
Deleted

Reply:
Sorry, I was not clear enough. I meant that this patch could delete TaskGroupSequentialFor test from unittests/Support/ParallelTest.cpp. As it was added for sequential mode.

Reply:
Thx. I just realized that build failure of the unittest. Deleted:)